yt_dlp/extractor/amcnetworks.py

```diff
@@ -26,6 +26,7 @@ class AMCNetworksIE(ThePlatformIE):  # XXX: Do not subclass from concrete IE
             # m3u8 download
             'skip_download': True,
         },
+        'skip': 'Returns 404 not available. BBCAmerica requires a sign-in for just about everything, so someone will have to find a link that works and set up a test for that.'
```

Comment:
Avoid extraneous details. Just `404 not available` is sufficient. Not deleting the test implies already that a replacement needs to be found in future.

Reply:
OK, will remove.

Reply:
@trainman261 did you forget about this?

Reply:
Yes, sorry, my bad.